README.rst

```diff
@@ -4,25 +4,73 @@ Astropy performance benchmarks
 About
 -----
 
-These benchmarks track the performance of various features in astropy
-*over time*.
+This repository includes a set of benchmarks to regularly test the performance of
+various parts of the astropy core package. These benchmarks are run for every
+new commit in the core package repository and the results are published using a
+user-friendly `web interface <http://astropy.org/astropy-benchmarks>`__. The
+benchmarks are run using `airspeed velocity <https://asv.readthedocs.io>`__ on
+dedicated physical machines belonging to members of the Astropy developer community.
 
-To view them, visit `this site
-<http://astropy.org/astropy-benchmarks>`__.
+Running the benchmarks locally
+------------------------------
 
-The benchmarks are run using `airspeed velocity
-<https://asv.readthedocs.io>`__.
+If you want to try and run the benchmarks locally you will first need to install asv::
 
-Contributing a benchmark
-------------------------
+    pip install asv
+
+then clone the benchmarks repository::
+
+    git clone git@github.com:astropy/astropy-benchmarks.git
```

Comment:
you may want to mention the `--single-branch` cloning option in case someone only wants to run some benchmarks locally without reporting the results, etc.

Reply:
Done